flocx_market/conf/flask.py

```diff
@@ -0,0 +1,19 @@
+from oslo_config import cfg
+
+
+opts = [
+    cfg.StrOpt('SQLALCHEMY_DATABASE_URI',
+               default='mysql+pymysql://flocx_market:qwerty123@127.0.0.1:3306/flocx_market'),
```

Comment:
I don't think we actually need this; including oslo.db in your config generator means that there'll be a [database] section that has a connection parameter.

Also, this sort of parameter probably should not have a default at all.